wls-basisdaten-service/src/test/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/configuration/UnicodeConfigurationTest.java

```diff
@@ -49,28 +54,36 @@ class UnicodeConfigurationTest {
     private TestRestTemplate testRestTemplate;
 
     @Autowired
-    private TheEntityRepository theEntityRepository;
+    private WahlRepository wahlRepository;
+
+    @Autowired
+    WahltagRepository wahltagRepository;
 
     @Test
-    @Disabled
     void testForNfcNormalization() {
-        // Persist entity with decomposed string.
-        final TheEntityDto theEntityDto = new TheEntityDto();
-        theEntityDto.setTextAttribute(TEXT_ATTRIBUTE_DECOMPOSED);
-        assertEquals(TEXT_ATTRIBUTE_DECOMPOSED.length(), theEntityDto.getTextAttribute().length());
-        final TheEntityDto response = testRestTemplate.postForEntity(URI.create(ENTITY_ENDPOINT_URL), theEntityDto, TheEntityDto.class).getBody();
 
-        // Check whether response contains a composed string.
-        assertEquals(TEXT_ATTRIBUTE_COMPOSED, response.getTextAttribute());
-        assertEquals(TEXT_ATTRIBUTE_COMPOSED.length(), response.getTextAttribute().length());
+        // Persist entity with decomposed string
+        // wahltag required for next step to store a list of wahlen
+        var searchingForWahltag = new Wahltag("wahltagID", LocalDate.now(), "beschreibung5", "1");
+        wahltagRepository.save(searchingForWahltag);
+        // create a list of Wahl with only one Wahl containing the TEXT_ATTRIBUTE_DECOMPOSED as 'name'
+        val wahlDTOList = createControllerListOfWahlDTO(searchingForWahltag);
+        assertEquals(TEXT_ATTRIBUTE_DECOMPOSED.length(), wahlDTOList.get(0).name().length());
+        // store list of Wahl
+        testRestTemplate.postForEntity(URI.create(WAHLEN_ENDPOINT_URL + searchingForWahltag.getWahltagID()), wahlDTOList, Void.class);
+
+        // Get the one and only Wahl from repo which now should contain a composed string in the 'name' attribute
+        val wahl = wahlRepository.findById("wahlID1").orElseThrow();
+        assertEquals(TEXT_ATTRIBUTE_COMPOSED, wahl.getName());
```

Comment:
Ich habe bisher in den neuen Services gesehen, dass immer AssertJ statt JUnit verwendet wurde. Also `Assertions.assertThat().isEqualTo();` statt `assertEquals()`

Ist das noch ein Element aus dem alten Code, oder ist das egal welche Bibliothek eingesetzt wird?